deploy/kubernetes/postee.yaml

```diff
@@ -192,9 +192,6 @@ spec:
               containerPort: 8445
               protocol: TCP
           volumeMounts:
-            - name: configmap-vol
-              mountPath: /configmap
-              readOnly: false
```

Comment:
Why did you remove this?

Reply:
In the initContainers/ setting-cfg we already copy the configmap, in Poste we are using the volume postee-config in the path /config